src/transformers/models/realm/utils_realm.py

```diff
@@ -0,0 +1,69 @@
+# coding=utf-8
```

Comment:
This file can now be completely removed. 

`ScaNNSearcher` and `convert_tfrecord_to_np` are in `realm_retrieval and the brute force searcher is for now embedded in the ForQA model.